docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md

```diff
@@ -23,11 +23,11 @@ lang: en
 <!-- Basic usage of the command -->
 <pre class="codeblock-pre">
   <div class="codeblock">
-    mesheryctl system status [flags]
+    mesheryctl system status 
   </div>
 </pre>
 
-## Examples
+<!--## Examples
```

Comment:
@alphaX86 I think `mesheryctl system status` requires examples right? 😶 

// @navendu-pottekkat

Reply:
@alphaX86 The title contains, mesheryctl system status | meshery. Can you please remove `| meshery` in this?

Also, this needs to be changed in all the files so title is without `| meshery`
![Screenshot from 2021-06-19 20-08-38](https://user-images.githubusercontent.com/23498248/122645860-2b517880-d13a-11eb-898b-7c4bdab1c4a4.png)

Reply:
It'd be repeated if I bring examples again

Reply:
I can change the title but regarding that example thing, I really doubt considering UX issues

Reply:
Tags removed. We need to decide if this change is good